electric_field_due_to_single_charge.py

```diff
@@ -0,0 +1,72 @@
+"""
+This module calculates the electric field due to a single charge and the net electric field
+due to multiple point charges using Coulomb's law.
+"""
+
+import numpy as np
+
+# Coulomb constant in Nm^2/C^2
+k = 8.99e9
+
+# Lambda functions
+separation_vector = lambda p, o: (p[0] - o[0], p[1] - o[1])  # Separation vector between two points
```

Comment:
Agreeing here with the linter:
electric_field_due_to_single_charge.py:12:20: C3001: Lambda expression assigned to a variable. Define a function using the "def" keyword instead. (unnecessary-lambda-assignment)
just use a function here